sass/directives/_cards.scss

```diff
@@ -0,0 +1,112 @@
+@mixin card--tile($scale: true, $include_strip: true) {
+  @include component-box;
+  @include display(flex);
+  @include flex-direction(column);
+  @include align-items(center);
+  @include flex(1);
+
+  @if $scale {
+    @include transform(scale(1));
+    @include transition(transform 0.4s ease-in);
+  }
+
+  @if $include_strip {
+    padding-top: calc(#{$base-spacing-medium} + #{$small-strip-height});
+
+    &:before {
+      @include bs__small-strip;
+    }
+  }
+
+  margin: $base-spacing-medium / 2;
+  text-align: center;
+  font-weight: $font-weight-normal;
+
+  &:hover,
+  &:active,
+  &:focus {
+    text-decoration: none;
+    box-shadow: $base-box-shadow-hover;
+  }
+
+  h3 {
+    text-transform: uppercase;
+    font-size: $small-font-size;
+  }
+
+  @media only screen and (min-width: $small-screen-max) { // TODO turning off the fanciness on mobile but should there still be some sort of state for usability?
+    &:hover {
+      @if $scale {
+        @include transform(scale(1.06));
+        @include transition(transform 550ms ease-out);
+      }
+
+      @if $include_strip {
+        &:before {
+          background-color: $anchor-blue;
+        }
+      }
+    }
+  }
+
+  @media only screen and (max-width: $medium-screen-max) {
+    @include flex(1 0 46%);
+  }
+
+  @media only screen and (max-width: $small-screen-max) {
+    width: 100%;
+    @include flex(1);
+    margin-left: 0;
+    margin-right: 0;
+
+    &:first-child {
+      margin-top: 0;
+    }
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+}
+
+@mixin card--tile__logo {
```

Comment:
@mrfantasticwonders try to use BEM. When naming one "block" or one "idea" I use single hyphens. In some places Rails requires underscores and no hyphens so in that case I use single underscores. 

So for a Small Business Page FAQ section that has some special modification class I'd have

`small-biz__faq--special`

If that thing were a Rails partial I'd have

`small_biz__faq--special`

Where "small biz" is one name or block, FAQ is a block/element/component within that name, and "special" is the modifier

So I'm using single hyphens and underscores for single ideas and separating each idea within a name using BEM syntax

Also, some of the naming you'll see might be old and can be refactored